### PR TITLE
Add Kik Messenger

### PR DESCRIPTION
**Pull request by Nicos18**

### CODE REVIEW

File: _data/communication.yml [hunk 1]
```diff
@@ -98,6 +98,13 @@ websites:
       facebook: irccloud
       tfa: No
 
+    - name: Kik Messenger
```
Comment (Carlgo11) [inline _data/communication.yml]: Maybe just let the name be `Kik`? Seems like they use that themselves.

File: _data/communication.yml [hunk 2]
```diff
@@ -98,6 +98,13 @@ websites:
...
+      url: https://www.kik.com/
+      img: kikmessenger.png
+      tfa: No
+      twitter: kik
+      facebook: kikplaynice
+
```
Comment (Carlgo11) [inline _data/communication.yml]: Add `email_address: support@kik.com`

File: _data/communication.yml [hunk 3]
```diff
@@ -98,6 +98,13 @@ websites:
       facebook: irccloud
       tfa: No
 
+    - name: Kik Messenger
+      url: https://www.kik.com/
+      img: kikmessenger.png
```
Comment (Carlgo11) [inline _data/communication.yml]: Same here, name could just be kik.

File: _data/communication.yml [hunk 4]
```diff
@@ -98,6 +98,13 @@ websites:
...
+      tfa: No
+      twitter: kik
```
Comment (Carlgo11) [inline _data/communication.yml]: Make the first K capital. (aka `twitter: Kik`)

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Please use this logo instead. It's smaller and transparent.
kik

**Review by Carlgo11 — APPROVED** [review]

File: _data/communication.yml [hunk 5]
```diff
@@ -98,6 +98,15 @@ websites:
       facebook: irccloud
       tfa: No
 
+
```
Comment (Jawshy) [inline _data/communication.yml:101]: There's an extra line break [here](https://github.com/2factorauth/twofactorauth/blob/7ea0c6a695238945d5d5f24bf3c56d7436eb3618/_data/communication.yml#L101). Could a pull request be skipped and go straight to committing to `master`, @2factorauth/core?

**Pull request merged**